docs/preview/connected-registry/release-notes.md

```diff
@@ -2,6 +2,15 @@
 
 Release notes for the Azure Container Registry connected registry runtime image. The image is published at `mcr.microsoft.com/acr/connected-registry:<tag>`.
 
+## 0.8.0
+July 29, 2022
+
+* Bug fix for memory leak on connected registry instance.
+* Support sync transfer chunk size as helm chart value.
```

Comment:
@savaradh we can remove this detail here since it is not relevant to the 0.8.0 image.

Reply:
@jaysterp removed it